pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -273,7 +289,7 @@ func (i *Instances) List(filter string) ([]string, error) {
 		return nil, err
 	}
 
-	glog.V(3).Infof("Found %s instances matching %s: %s",
+	glog.V(3).Infof("found %s instances matching %s: %s",
```

Comment:
You can leave this alone -- I gave you bad advice before re: log messages starting with lowercase.  My bad.